app/src/main/java/com/randomappsinc/studentpicker/home/DeleteListDialog.java

```diff
@@ -19,16 +19,21 @@
     DeleteListDialog(@NonNull Listener listener, Context context) {
         dialog = new MaterialDialog.Builder(context)
                 .title(R.string.confirm_deletion_title)
-                .content(R.string.confirm_deletion_message)
                 .positiveText(android.R.string.yes)
                 .negativeText(android.R.string.cancel)
                 .onPositive((dialog, which) ->
                         listener.onDeleteListConfirmed(position)
                 ).build();
     }
 
-    public void show(int position) {
+    // Well there is no purpose of sending the position here do you keep it for the upcoming work?
```

Comment:
I'm glad you're asking these questions! For the future, questions like these are better left as comments on your own pull request instead of in the code (since you will have to delete them later on).

The position is unfortunately a bit of a hack so the activity knows which item to update in the adapter once the action is taken.